src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -511,7 +511,7 @@ void ShenandoahHeap::initialize_heuristics_generations() {
   // for old would be total heap - minimum capacity of young. This means the sum of the maximum
   // allowed for old and young could exceed the total heap size. It remains the case that the
   // _actual_ capacity of young + old = total.
```

Comment:
Not your change, but it seems like this comment might belong in the _declaration_ of `max_capacity()` (or of `_max_capacity`), rather than here?

Reply:
I believe this code will also be replaced soon.